docs/snippets/prerequisites.md

```diff
@@ -11,7 +11,7 @@ Before installing Knative, you must meet the following prerequisites:
 - **For production purposes**, it is recommended that:
 
     - If you have only one node in your cluster, you need at least 6&nbsp;CPUs, 6&nbsp;GB of memory, and 30&nbsp;GB of disk storage.
-    - If you have multiple nodes in your cluster, for each node you need at least 2&nbsp;CPUs, 4&nbsp;GB of memory, and 20&nbsp;GB of disk storage.
+    - If you have multiple nodes in your cluster, for each node you need at least 3&nbsp;CPUs, 4&nbsp;GB of memory, and 20&nbsp;GB of disk storage.
```

Comment:
I think making this update for production too is probably the right choice, but let's confirm with @csantanapr

Reply:
You can leave it at 2CPUs, since the 2CPUs is per worker and this line is about multiple nodes, so at the minimum cluster will have 4 CPUs in total.

Reply:
TLDR; don't change this line